infra/terraform/modules/service/variables.tf

```diff
@@ -40,3 +40,14 @@ variable "services" {
   description = "The services to deploy"
   default     = {}
 }
+
+variable "job_definitions" {
+  type = map(object({
+    job_name = string
+    command  = list(string)
+    image    = string
+    memory   = string
```

Comment:
Would `number` be more appropriate here?

Reply:
Yep but the terraform failed when it tried to convert it to json